### ISSUE TITLE
problem regarding depth_refinement_loss 

### ISSUE THREAD

**Issue by bpwl0121** [Brummi/MonoRec#27]
Hi Felix,

https://github.com/Brummi/MonoRec/blob/b2b3decc130d9ca333d1350096b9838a12f977a3/model/loss_functions/monorec_loss.py#L334
https://github.com/Brummi/MonoRec/blob/b2b3decc130d9ca333d1350096b9838a12f977a3/model/loss_functions/monorec_loss.py#L339
you use the .detach() method in the sdl at depth_refinement_loss, which cut the backpropagation, different from the depth_refinement_loss in the paper. Or did I misunderstand?

**Comment by Brummi** [comment]
Hi bpwl0121,

thank you for looking into the code so thoroughly, I appreciate it a lot!
I double checked and the code I used for the paper corresponds to the version in the paper. 

But you seem to be correct here, I will look into it.
I suspect I may have introduced a mistake when refactoring the code before publishing it.

Best,
Felix
